Added the hyperlinks

Debanitrkl opened the pull request:
Added the hyperlinks to the next steps

Ompragash reviewed (changes requested):
@Debanitrkl I don't see all the files under `docs/sub-folders` 
- [ ] any reason why you didn't include them?
- [ ] Let's keep all the files as is, and prefix them with a proper number system eg: all unique topics can have a prefix 01, 02, 03 and if 03 has sub-docs then name them like 031, 032

Debanitrkl commented:
> * any reason why you didn't include them?

So for the rest of the sub-folders remaining contains the docs that are pre-requisites for running the pipeline like having a cluster and delegate, the reason keeping it in sub-folders is those can be achieved in many ways and for now we have just added few examples which could be further extended by community contribution like we have for the azure infra for delegates, so I think keeping those separate under a sub-directory will help and rest about adding numbers to it, I'm bit confused on how to number those steps as they can be done in any order nothing order specific, the only that could be ordered was creating the pipeline and going all the way to the execution steps hence I removed it out of sub-folders and numbered it.
